Add a way to disable output

Hi! I am using this tool in my project and I figured out that in some tests it generates quite a lot of output.

Disabling this tool right now does not seem very easy, because I can't modify `MIDDLEWARE` when apps are already loaded with `pytest-django`, so I would have to write a plugin for this.

Maybe a settings like `DQC_ENABLE = True`, which can be set to `False` when needed, can disable the whole output?

This is what I have currently:

```
---------------------------------- Captured stdout setup ----------------------------------
+--------+--------+--------+--------+------------+-------+----------+
| Select | Insert | Update | Delete | Duplicates | Total | Duration |
+--------+--------+--------+--------+------------+-------+----------+
|   0    |   0    |   0    |   0    |     0      |   0   |   0.04   |
+--------+--------+--------+--------+------------+-------+----------+
Target: /docs/openapi.json/ dmr.openapi.views.json.OpenAPIJsonView
---------------------------------- Captured stdout call -----------------------------------
+--------+--------+--------+--------+------------+-------+----------+
| Select | Insert | Update | Delete | Duplicates | Total | Duration |
+--------+--------+--------+--------+------------+-------+----------+
|   0    |   0    |   0    |   0    |     0      |   0   |   0.00   |
+--------+--------+--------+--------+------------+-------+----------+
Target: /api/user/users/ server.apps.main.api.views.BlogPostCreate
+--------+--------+--------+--------+------------+-------+----------+
| Select | Insert | Update | Delete | Duplicates | Total | Duration |
+--------+--------+--------+--------+------------+-------+----------+
|   0    |   0    |   0    |   0    |     0      |   0   |   0.00   |
+--------+--------+--------+--------+------------+-------+----------+
Target: /api/user/users/ server.apps.main.api.views.BlogPostCreate
+--------+--------+--------+--------+------------+-------+----------+
| Select | Insert | Update | Delete | Duplicates | Total | Duration |
+--------+--------+--------+--------+------------+-------+----------+
|   0    |   0    |   0    |   0    |     0      |   0   |   0.00   |
+--------+--------+--------+--------+------------+-------+----------+
Target: /api/user/users/ server.apps.main.api.views.BlogPostCreate
+--------+--------+--------+--------+------------+-------+----------+
| Select | Insert | Update | Delete | Duplicates | Total | Duration |
+--------+--------+--------+--------+------------+-------+----------+
|   0    |   0    |   0    |   0    |     0      |   0   |   0.00   |
+--------+--------+--------+--------+------------+-------+----------+
Target: /api/user/users/ server.apps.main.api.views.BlogPostCreate
+--------+--------+--------+--------+------------+-------+----------+
| Select | Insert | Update | Delete | Duplicates | Total | Duration |
+--------+--------+--------+--------+------------+-------+----------+
|   0    |   0    |   0    |   0    |     0      |   0   |   0.00   |
+--------+--------+--------+--------+------------+-------+----------+
Target: /api/user/users/ server.apps.main.api.views.BlogPostCreate
+--------+--------+--------+--------+------------+-------+----------+
| Select | Insert | Update | Delete | Duplicates | Total | Duration |
+--------+--------+--------+--------+------------+-------+----------+
|   0    |   0    |   0    |   0    |     0      |   0   |   0.00   |
+--------+--------+--------+--------+------------+-------+----------+
Target: /api/user/users/ server.apps.main.api.views.BlogPostCreate
+--------+--------+--------+--------+------------+-------+----------+
| Select | Insert | Update | Delete | Duplicates | Total | Duration |
+--------+--------+--------+--------+------------+-------+----------+
|   0    |   0    |   0    |   0    |     0      |   0   |   0.00   |
+--------+--------+--------+--------+------------+-------+----------+
Target: /api/user/users/ server.apps.main.api.views.BlogPostCreate
+--------+--------+--------+--------+------------+-------+----------+
| Select | Insert | Update | Delete | Duplicates | Total | Duration |
+--------+--------+--------+--------+------------+-------+----------+
|   0    |   0    |   0    |   0    |     0      |   0   |   0.00   |
+--------+--------+--------+--------+------------+-------+----------+
Target: /api/user/users/ server.apps.main.api.views.BlogPostCreate
+--------+--------+--------+--------+------------+-------+----------+
| Select | Insert | Update | Delete | Duplicates | Total | Duration |
+--------+--------+--------+--------+------------+-------+----------+
|   0    |   0    |   0    |   0    |     0      |   0   |   0.00   |
+--------+--------+--------+--------+------------+-------+----------+
Target: /api/user/users/ server.apps.main.api.views.BlogPostCreate
+--------+--------+--------+--------+------------+-------+----------+
| Select | Insert | Update | Delete | Duplicates | Total | Duration |
+--------+--------+--------+--------+------------+-------+----------+
|   0    |   0    |   0    |   0    |     0      |   0   |   0.00   |
+--------+--------+--------+--------+------------+-------+----------+
Target: /api/user/users/ server.apps.main.api.views.BlogPostCreate
+--------+--------+--------+--------+------------+-------+----------+
| Select | Insert | Update | Delete | Duplicates | Total | Duration |
+--------+--------+--------+--------+------------+-------+----------+
|   0    |   0    |   0    |   0    |     0      |   0   |   0.00   |
+--------+--------+--------+--------+------------+-------+----------+
Target: /api/user/users/ server.apps.main.api.views.BlogPostCreate
+--------+--------+--------+--------+------------+-------+----------+
| Select | Insert | Update | Delete | Duplicates | Total | Duration |
+--------+--------+--------+--------+------------+-------+----------+
|   0    |   0    |   0    |   0    |     0      |   0   |   0.00   |
+--------+--------+--------+--------+------------+-------+----------+
Target: /api/user/users/ server.apps.main.api.views.BlogPostCreate
+--------+--------+--------+--------+------------+-------+----------+
| Select | Insert | Update | Delete | Duplicates | Total | Duration |
+--------+--------+--------+--------+------------+-------+----------+
|   0    |   0    |   0    |   0    |     0      |   0   |   0.00   |
+--------+--------+--------+--------+------------+-------+----------+
Target: /api/user/users/ server.apps.main.api.views.BlogPostCreate
+--------+--------+--------+--------+------------+-------+----------+
| Select | Insert | Update | Delete | Duplicates | Total | Duration |
+--------+--------+--------+--------+------------+-------+----------+
|   0    |   0    |   0    |   0    |     0      |   0   |   0.00   |
+--------+--------+--------+--------+------------+-------+----------+
Target: /api/user/users/ server.apps.main.api.views.BlogPostCreate
+--------+--------+--------+--------+------------+-------+----------+
| Select | Insert | Update | Delete | Duplicates | Total | Duration |
+--------+--------+--------+--------+------------+-------+----------+
|   0    |   0    |   0    |   0    |     0      |   0   |   0.00   |
+--------+--------+--------+--------+------------+-------+----------+
Target: /api/user/users/ server.apps.main.api.views.BlogPostCreate
+--------+--------+--------+--------+------------+-------+----------+
| Select | Insert | Update | Delete | Duplicates | Total | Duration |
+--------+--------+--------+--------+------------+-------+----------+
|   0    |   0    |   0    |   0    |     0      |   0   |   0.00   |
+--------+--------+--------+--------+------------+-------+----------+
Target: /api/user/users/ server.apps.main.api.views.BlogPostCreate
+--------+--------+--------+--------+------------+-------+----------+
| Select | Insert | Update | Delete | Duplicates | Total | Duration |
+--------+--------+--------+--------+------------+-------+----------+
|   0    |   0    |   0    |   0    |     0      |   0   |   0.00   |
+--------+--------+--------+--------+------------+-------+----------+
Target: /api/user/users/ server.apps.main.api.views.BlogPostCreate
+--------+--------+--------+--------+------------+-------+----------+
| Select | Insert | Update | Delete | Duplicates | Total | Duration |
+--------+--------+--------+--------+------------+-------+----------+
|   0    |   0    |   0    |   0    |     0      |   0   |   0.00   |
+--------+--------+--------+--------+------------+-------+----------+
Target: /api/user/users/ server.apps.main.api.views.BlogPostCreate
+--------+--------+--------+--------+------------+-------+----------+
| Select | Insert | Update | Delete | Duplicates | Total | Duration |
+--------+--------+--------+--------+------------+-------+----------+
|   0    |   0    |   0    |   0    |     0      |   0   |   0.00   |
+--------+--------+--------+--------+------------+-------+----------+
Target: /api/user/users/ server.apps.main.api.views.BlogPostCreate
+--------+--------+--------+--------+------------+-------+----------+
| Select | Insert | Update | Delete | Duplicates | Total | Duration |
+--------+--------+--------+--------+------------+-------+----------+
|   0    |   0    |   0    |   0    |     0      |   0   |   0.00   |
+--------+--------+--------+--------+------------+-------+----------+
Target: /api/user/users/ server.apps.main.api.views.BlogPostCreate
+--------+--------+--------+--------+------------+-------+----------+
| Select | Insert | Update | Delete | Duplicates | Total | Duration |
+--------+--------+--------+--------+------------+-------+----------+
|   0    |   0    |   0    |   0    |     0      |   0   |   0.00   |
+--------+--------+--------+--------+------------+-------+----------+
Target: /api/user/users/ server.apps.main.api.views.BlogPostCreate
+--------+--------+--------+--------+------------+-------+----------+
| Select | Insert | Update | Delete | Duplicates | Total | Duration |
+--------+--------+--------+--------+------------+-------+----------+
|   0    |   0    |   0    |   0    |     0      |   0   |   0.00   |
+--------+--------+--------+--------+------------+-------+----------+
Target: /api/user/users/ server.apps.main.api.views.BlogPostCreate
+--------+--------+--------+--------+------------+-------+----------+
| Select | Insert | Update | Delete | Duplicates | Total | Duration |
+--------+--------+--------+--------+------------+-------+----------+
|   0    |   0    |   0    |   0    |     0      |   0   |   0.00   |
+--------+--------+--------+--------+------------+-------+----------+
Target: /api/user/users/ server.apps.main.api.views.BlogPostCreate
+--------+--------+--------+--------+------------+-------+----------+
| Select | Insert | Update | Delete | Duplicates | Total | Duration |
+--------+--------+--------+--------+------------+-------+----------+
|   0    |   0    |   0    |   0    |     0      |   0   |   0.00   |
+--------+--------+--------+--------+------------+-------+----------+
Target: /api/user/users/ server.apps.main.api.views.BlogPostCreate
+--------+--------+--------+--------+------------+-------+----------+
| Select | Insert | Update | Delete | Duplicates | Total | Duration |
+--------+--------+--------+--------+------------+-------+----------+
|   0    |   0    |   0    |   0    |     0      |   0   |   0.00   |
+--------+--------+--------+--------+------------+-------+----------+
Target: /api/user/users/ server.apps.main.api.views.BlogPostCreate
+--------+--------+--------+--------+------------+-------+----------+
| Select | Insert | Update | Delete | Duplicates | Total | Duration |
+--------+--------+--------+--------+------------+-------+----------+
|   0    |   0    |   0    |   0    |     0      |   0   |   0.00   |
+--------+--------+--------+--------+------------+-------+----------+
Target: /api/user/users/ server.apps.main.api.views.BlogPostCreate
+--------+--------+--------+--------+------------+-------+----------+
| Select | Insert | Update | Delete | Duplicates | Total | Duration |
+--------+--------+--------+--------+------------+-------+----------+
|   0    |   1    |   0    |   0    |     0      |   1   |   0.01   |
+--------+--------+--------+--------+------------+-------+----------+
Target: /api/user/users/ server.apps.main.api.views.BlogPostCreate
+--------+--------+--------+--------+------------+-------+----------+
| Select | Insert | Update | Delete | Duplicates | Total | Duration |
+--------+--------+--------+--------+------------+-------+----------+
|   0    |   1    |   0    |   0    |     0      |   1   |   0.01   |
+--------+--------+--------+--------+------------+-------+----------+
Target: /api/user/users/ server.apps.main.api.views.BlogPostCreate
+--------+--------+--------+--------+------------+-------+----------+
| Select | Insert | Update | Delete | Duplicates | Total | Duration |
+--------+--------+--------+--------+------------+-------+----------+
|   0    |   1    |   0    |   0    |     0      |   1   |   0.01   |
+--------+--------+--------+--------+------------+-------+----------+
Target: /api/user/users/ server.apps.main.api.views.BlogPostCreate
+--------+--------+--------+--------+------------+-------+----------+
| Select | Insert | Update | Delete | Duplicates | Total | Duration |
+--------+--------+--------+--------+------------+-------+----------+
|   0    |   1    |   0    |   0    |     0      |   1   |   0.01   |
+--------+--------+--------+--------+------------+-------+----------+
Target: /api/user/users/ server.apps.main.api.views.BlogPostCreate
+--------+--------+--------+--------+------------+-------+----------+
| Select | Insert | Update | Delete | Duplicates | Total | Duration |
+--------+--------+--------+--------+------------+-------+----------+
|   0    |   1    |   0    |   0    |     0      |   1   |   0.01   |
+--------+--------+--------+--------+------------+-------+----------+
Target: /api/user/users/ server.apps.main.api.views.BlogPostCreate
+--------+--------+--------+--------+------------+-------+----------+
| Select | Insert | Update | Delete | Duplicates | Total | Duration |
+--------+--------+--------+--------+------------+-------+----------+
|   0    |   1    |   0    |   0    |     0      |   1   |   0.01   |
+--------+--------+--------+--------+------------+-------+----------+
Target: /api/user/users/ server.apps.main.api.views.BlogPostCreate
+--------+--------+--------+--------+------------+-------+----------+
| Select | Insert | Update | Delete | Duplicates | Total | Duration |
+--------+--------+--------+--------+------------+-------+----------+
|   0    |   1    |   0    |   0    |     0      |   1   |   0.01   |
+--------+--------+--------+--------+------------+-------+----------+
Target: /api/user/users/ server.apps.main.api.views.BlogPostCreate
__________________ test_schemathesis (label='GET /api/user/users/{id}') ___________________
.venv/lib/python3.13/site-packages/schemathesis/generation/hypothesis/builder.py:268: in test_schemathesis
    def test_wrapper(*args: Any, **kwargs: Any) -> Any:
               ^^^^^^^^^^^
.venv/lib/python3.13/site-packages/hypothesis/core.py:1656: in _raise_to_user
    raise the_error_hypothesis_found
tests/test_apps/test_schema.py:45: in test_schemathesis
    assert False
E   assert False
E   (note: 12 other explicit examples also failed with this error; use Verbosity.verbose to view)
---------------------------------- Captured stdout setup ----------------------------------
+--------+--------+--------+--------+------------+-------+----------+
| Select | Insert | Update | Delete | Duplicates | Total | Duration |
+--------+--------+--------+--------+------------+-------+----------+
|   0    |   0    |   0    |   0    |     0      |   0   |   0.04   |
+--------+--------+--------+--------+------------+-------+----------+
Target: /docs/openapi.json/ dmr.openapi.views.json.OpenAPIJsonView
---------------------------------- Captured stdout call -----------------------------------
+--------+--------+--------+--------+------------+-------+----------+
| Select | Insert | Update | Delete | Duplicates | Total | Duration |
+--------+--------+--------+--------+------------+-------+----------+
|   0    |   0    |   0    |   0    |     0      |   0   |   0.00   |
+--------+--------+--------+--------+------------+-------+----------+
Target: /api/user/users/0 server.apps.main.api.views.BlogPostGet
+--------+--------+--------+--------+------------+-------+----------+
| Select | Insert | Update | Delete | Duplicates | Total | Duration |
+--------+--------+--------+--------+------------+-------+----------+
|   0    |   0    |   0    |   0    |     0      |   0   |   0.00   |
+--------+--------+--------+--------+------------+-------+----------+
Target: /api/user/users/0 server.apps.main.api.views.BlogPostGet
+--------+--------+--------+--------+------------+-------+----------+
| Select | Insert | Update | Delete | Duplicates | Total | Duration |
+--------+--------+--------+--------+------------+-------+----------+
|   0    |   0    |   0    |   0    |     0      |   0   |   0.00   |
+--------+--------+--------+--------+------------+-------+----------+
Target: /api/user/users/0 server.apps.main.api.views.BlogPostGet
+--------+--------+--------+--------+------------+-------+----------+
| Select | Insert | Update | Delete | Duplicates | Total | Duration |
+--------+--------+--------+--------+------------+-------+----------+
|   0    |   0    |   0    |   0    |     0      |   0   |   0.00   |
+--------+--------+--------+--------+------------+-------+----------+
Target: /api/user/users/0 server.apps.main.api.views.BlogPostGet
+--------+--------+--------+--------+------------+-------+----------+
| Select | Insert | Update | Delete | Duplicates | Total | Duration |
+--------+--------+--------+--------+------------+-------+----------+
|   0    |   0    |   0    |   0    |     0      |   0   |   0.00   |
+--------+--------+--------+--------+------------+-------+----------+
Target: /api/user/users/0 server.apps.main.api.views.BlogPostGet
+--------+--------+--------+--------+------------+-------+----------+
| Select | Insert | Update | Delete | Duplicates | Total | Duration |
+--------+--------+--------+--------+------------+-------+----------+
|   0    |   0    |   0    |   0    |     0      |   0   |   0.00   |
+--------+--------+--------+--------+------------+-------+----------+
Target: /api/user/users/0 server.apps.main.api.views.BlogPostGet
+--------+--------+--------+--------+------------+-------+----------+
| Select | Insert | Update | Delete | Duplicates | Total | Duration |
+--------+--------+--------+--------+------------+-------+----------+
|   0    |   0    |   0    |   0    |     0      |   0   |   0.00   |
+--------+--------+--------+--------+------------+-------+----------+
Target: /api/user/users/0 server.apps.main.api.views.BlogPostGet
+--------+--------+--------+--------+------------+-------+----------+
| Select | Insert | Update | Delete | Duplicates | Total | Duration |
+--------+--------+--------+--------+------------+-------+----------+
|   0    |   0    |   0    |   0    |     0      |   0   |   0.00   |
+--------+--------+--------+--------+------------+-------+----------+
Target: /api/user/users/null,null
+--------+--------+--------+--------+------------+-------+----------+
| Select | Insert | Update | Delete | Duplicates | Total | Duration |
+--------+--------+--------+--------+------------+-------+----------+
|   0    |   0    |   0    |   0    |     0      |   0   |   0.00   |
+--------+--------+--------+--------+------------+-------+----------+
Target: /api/user/users/¡;ÿ
+--------+--------+--------+--------+------------+-------+----------+
| Select | Insert | Update | Delete | Duplicates | Total | Duration |
+--------+--------+--------+--------+------------+-------+----------+
|   0    |   0    |   0    |   0    |     0      |   0   |   0.00   |
+--------+--------+--------+--------+------------+-------+----------+
Target: /api/user/users/null
+--------+--------+--------+--------+------------+-------+----------+
| Select | Insert | Update | Delete | Duplicates | Total | Duration |
+--------+--------+--------+--------+------------+-------+----------+
|   0    |   0    |   0    |   0    |     0      |   0   |   0.00   |
+--------+--------+--------+--------+------------+-------+----------+
Target: /api/user/users/false
+--------+--------+--------+--------+------------+-------+----------+
| Select | Insert | Update | Delete | Duplicates | Total | Duration |
+--------+--------+--------+--------+------------+-------+----------+
|   0    |   0    |   0    |   0    |     0      |   0   |   0.00   |
+--------+--------+--------+--------+------------+-------+----------+
Target: /api/user/users/3.9229476907851786e-263
+--------+--------+--------+--------+------------+-------+----------+
| Select | Insert | Update | Delete | Duplicates | Total | Duration |
+--------+--------+--------+--------+------------+-------+----------+
|   1    |   0    |   0    |   0    |     0      |   1   |   0.01   |
+--------+--------+--------+--------+------------+-------+----------+
Target: /api/user/users/0 server.apps.main.api.views.BlogPostGet
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,5 +1,9 @@
 # Changelog
 
+## 0.5.0
+
+- added DQC_ENABLED setting to disable output (#24)
+
 ## 0.4.2
 
 - added check for django6/py3.14

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -10,7 +10,7 @@ packages = ["query_counter"]
 
 [project]
 name = "django-query-counter"
-version = "0.4.2"
+version = "0.5.0"
 description = "Debug tool to print SQL queries count to the console"
 authors = [{name = "Oleg Smedyuk", email = "oleg.smedyuk@gmail.com"}]
 license = "MIT"

--- a/query_counter/decorators.py
+++ b/query_counter/decorators.py
@@ -176,6 +176,8 @@ class QueryLogger:
 
 def queries_counter(func):
     def inner_func(*args, **kwargs):
+        if not _get_value('DQC_ENABLED'):
+            return func(*args, **kwargs)
         func_info = ['Target:']
         query_logger = QueryLogger()
         with connection.execute_wrapper(query_logger):

--- a/query_counter/settings.py
+++ b/query_counter/settings.py
@@ -1,4 +1,5 @@
 DEFAULTS = {
+    'DQC_ENABLED': True,
     'DQC_SLOWEST_COUNT': 5,
     'DQC_DUPLICATED_COUNT': 10,
     'DQC_TABULATE_FMT': 'pretty',

--- a/tests/test_main.py
+++ b/tests/test_main.py
@@ -17,3 +17,18 @@ def test_view_with_decorator(
     out, err = capsys.readouterr()
     assert 'Duplicate queries:' in out
     assert 'Target: / urls.index' in out
+
+
+def test_dqc_enabled_false_suppresses_output(
+    db, client, django_assert_num_queries, capsys, settings,
+):
+    settings.INSTALLED_APPS.append('query_counter')
+    settings.MIDDLEWARE.append('query_counter.middleware.DjangoQueryCounterMiddleware')
+    settings.DQC_ENABLED = False
+
+    with django_assert_num_queries(13):
+        response = client.get('')
+        assert response.status_code == 200
+
+    out, err = capsys.readouterr()
+    assert out == ''

